x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/left/components/host_details.tsx

```diff
@@ -147,6 +162,41 @@ export const HostDetails: React.FC<HostDetailsProps> = ({ hostName, timestamp, s
     skip: selectedPatterns.length === 0,
   });
 
+  const filterQuery = useMemo(
+    () => (hostName ? buildHostNamesFilter([hostName]) : undefined),
+    [hostName]
+  );
+  const { data: hostRisk } = useRiskScore({
+    filterQuery,
+    riskEntity: EntityType.host,
+    skip: hostName == null,
+    timerange,
+  });
+  const hostRiskData = hostRisk && hostRisk.length > 0 ? hostRisk[0] : undefined;
+  const isRiskScoreExist = !!hostRiskData?.host.risk;
+
+  const { hasNonClosedAlerts } = useNonClosedAlerts({
+    field: EntityIdentifierFields.hostName,
+    value: hostName,
+    to,
+    from,
+    queryId: 'HostEntityOverview',
+  });
+  const { hasMisconfigurationFindings } = useHasMisconfigurations('host.name', hostName);
+  const { hasVulnerabilitiesFindings } = useHasVulnerabilities('host.name', hostName);
+
+  const { openDetailsPanel } = useNavigateToHostDetails({
```

Comment:
What do you think about moving this `useNavigationToHostDetails` to the `shared` folder at the root of the `flyout` folder, now that it's used in both `document_details` and `entity_details` folders?

Also I feel like it would be a good idea to either add a new property or rename the `isPreviewMode` property to something else. It is a bit confusing to use `isPreviewMode` here (confusing enough that you had to add a code comment...)